edbterraform/lib.py

```diff
@@ -378,4 +350,4 @@ def spec_compatability(infrastructure_variables, cloud_service_provider):
             if 'zone_name' not in spec_variables['kubernetes'][cluster] and 'zone' in spec_variables['kubernetes'][cluster]:
                 spec_variables['kubernetes'][cluster]['zone_name'] = f'depreciated-{spec_variables["kubernetes"][machine]["zone"]}'
 
-    return spec_variables
+    return spec_variables    
```

Comment:
Please update remove the unnecessary spaces.

Reply:
https://github.com/EnterpriseDB/edb-terraform/pull/55/commits/7cfdb7627c81b45f2c1d7bd423051fdedfd67300